zer0-os/zUI
# feat(alert): info `Alert` icon

## 1. Pull request checklist



- [x] PR name follows [conventional commit spec](https://www.conventionalcommits.org/en/v1.0.0/)

## 2. What is the old behaviour? (if applicable)



- Info alerts would show a "!" icon (warning) instead of a "i" icon (info).
- The `IconInfoCircle` icon was outdated.

## 3. What is the new behaviour?



- Info alert uses "i".
- `IconInfoCircle` is up to date.

## 4. How can this behaviour be tested? (if applicable)

- Open Storybook to `Alert -> Info`.
- Check the component against Figma.

## 5. Other information
